src/docs/concepts-integrations/integrations/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Introduction to integrations in Flex
+slug: integrations-introduction
+updated: 2022-10-05
+category: concepts-integrations
+ingress:
+  Flex allows integrations to a vast range of third party tools and
+  solutions. These tools can be integrated to Flex by using Zapier, the
+  Flex Integration API, or the client application.
+published: true
+---
+
+Integrations in Flex refer to third-party solutions and tools that
+communicate with the Flex marketplace to either retrieve information and
+take action outside Flex, or trigger actions within Flex. A lot of
+features that are not natively available in Flex can be integrated to
+your marketplace solution with varying degrees of complexity.
+
+## Flex integration tools
+
+There are several ways to approach integrating third party tools to your
+Flex marketplace. Which tool you choose within Flex depends on the
+requirements of the third party tool you choose, and on the complexity
+of the integration you are looking to create.
+
+### Client application
+
+The simplest integration tool is your client application. Since you have
+full control over your marketplace client app, you can add analytics and
+tracking tools such as Hotjar, or messaging tools such as Sendbird or
+Intercom, directly into your client app code.
+
+The FTW templates include a Node.js/Express server by default. Some
+third party integrations require a secure context, or influence pricing,
+such as Voucherify or other promotion integrations. For these types of
+integrations, you can use the FTW server to create endpoints or other
+logic in a secure way.
+
+### Integration API
+
+If you want to create an integration where the third party solution
+listens to, or takes action on, the Flex marketplace data, you will
+likely need to use the Integration API. Where Marketplace API is meant
+for everyday marketplace interactions, Integration API provides
+operators powerful access to all marketplace data and activity. Through
+Integration API, operators can enable third party tools to create and
```

Comment:
Question: Are you not supposed to use a definite article before "Integration API" and "Marketplace API"?

Reply:
I guess that's a matter of preference. I think of them as proper names, in which case one wouldn't use an article, but I can double check the usage in our docs and add the articles if they're mostly used with articles.